gravitee-apim-console-webui/src/components/alerts/alertEnginePromotion/alert-engine-promotion.html

```diff
@@ -0,0 +1,50 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="ae-promo">
+  <div>
+    <div>
+      <h2 class="promo-title">Get started with Alerts</h2>
+    </div>
+    <div class="image-content">
+      <img src="../../../assets/api-observability-dark.png" alt="promotion" class="promotion">
```

Comment:
I wonder if the class name `promotion` should be something more precise.
Maybe like `promotion-image` ?

Reply:
done, updated class name to "promotion-image"